analyses/malawi/dryspells_trigger/exploration/mwi_viz_dry_spells_map.R

```diff
@@ -46,7 +46,7 @@ data <- mwi_adm2 %>%
   left_join(ds_counts, by = c("ADM2_PCODE" = "pcode"))
 
 # create map
-data %>%
+plt_ds_map <- data %>%
```

Comment:
If we assign the plot to a variable, I don't think the saving function ln 70 will work, since it saves the plot last generated in Viewer. Did you test it?

An option might be to assign it, then call `plt_ds_map` on ln 68 before it gets saved.

Reply:
Uhuh good catch. I tested it and it seems to work, but safer to indicate to ggsave that it is plt_ds_map that it should save, so added that now.